pages/api/users/add.ts

```diff
@@ -41,7 +42,38 @@ import { connectToDatabase } from '../../../Utils/mongodb';
  *        description: Created
  */
 
-export default async (req: any, res: any) => {
+enum Gender {
```

Comment:
@JRakin can we put gender in a golobal enum so that it can be refereed later for other pages